specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2022-08-03/gallery.json

```diff
@@ -742,7 +742,8 @@
             "type": "string",
             "description": "The expand expression to apply on the operation.",
             "enum": [
```

Comment:
There are 3 LintDiff errors that you should be able to fix, seen in this comment:
https://github.com/Azure/azure-rest-api-specs/pull/23507#issuecomment-1502556365

Reply:
fixed